pkgs/development/python-modules/torch/bin.nix

```diff
@@ -14,13 +15,18 @@
 , requests
 , setuptools
 , typing-extensions
+, sympy
+, jinja2
+, networkx
+, filelock
+, triton-bin
```

Comment:
H'm... it seems that other derivations (e.g. torchvision-bin) do it this way as well. Nonetheless, I'd rather declare the formal parameter with the source-build's name. I.e. I'd take `openai-triton`, but pass `openai-triton-bin` in the `callPackage`. This way the overrides to `torch` and `torch-bin` look exactly the same and I don't have to guess which naming scheme to use

Feel free to ignore this comment though